build/mkdocs-serve.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+# This script installs requirements for and starts the live-reloading development server for MkDocs.
+# MkDocs generates a static site from the markdown files in docs/.
+
+py_command=python
+pip_command=pip
+
+setup_py_command() {
+	if [ -z $(which ${py_command}) ]; then
+		py_command=python3
+		pip_command=pip3
+		return
+	fi
+	py_command_major_version=$(${py_command} 2>&1 --version | cut -d' ' -f2 | head -c1)
+	if [ $py_command_major_version != 3 ]; then
+		py_command=python3
+		pip_command=pip3
+	fi
+}
+setup_py_command
```

Comment:
This is kind of awkward; assumes that if `python` is v2 then `pip` will be too; assumes that if so, then `pip3` should be available. If `pip` or `pip3` is not will fail with a `command not found` error at https://github.com/etclabscore/core-geth/pull/294/files#diff-14af3ec200ad61f89a2faf7f748f9fc872bdb23f3836662f2b4a21e0108c6032R24.

There is probably a cleaner way of doing this (maybe even a conventional one?).

Reply:
You can call any python script using `python -m`, so for example `python -m pip --version`

Reply:
Resolved by https://github.com/etclabscore/core-geth/pull/294/commits/946fa8610b26cc4415b1cb632274c825c5e503b3